skaero/geometry/joukowsky.py

```diff
@@ -0,0 +1,21 @@
+import pylab as pl
```

Comment:
I would try to avoid pylab imports, and use `import numpy as np` and `import matplotlib.pyplot as plt` in this case instead.